policyengine_canada/parameters/gov/provinces/nu/tax/income/credits/age_amount/eligible_age.yaml

```diff
@@ -0,0 +1,9 @@
+description: Nunavut allows people who are over this age to receive the age amount credit. 
+values:
+  2023-01-01: 65
```

Comment:
```suggestion
  2002-01-01: 65
```

Reply:
Wondering why we are using 2002 here?

Reply:
The title of the law is `Age credit for 2002 and subsequent taxation years`